src/ResourceManager/DevTestLabs/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Current Release
 
+## Version 5.2.0
```

Comment:
This should go under current release, we will update the version tag at release time.

Reply:
I have removed the version tag.